app/sections/collection-header.tsx

```diff
@@ -0,0 +1,142 @@
+import type {
+    HydrogenComponentProps,
+    HydrogenComponentSchema,
+} from '@weaverse/hydrogen';
+import { forwardRef } from 'react';
+import { useLoaderData } from '@remix-run/react';
+import { CollectionDetailsQuery } from 'storefrontapi.generated';
+
+interface HeaderProps extends HydrogenComponentProps {
+    heightSectionDesktop: number;
```

Comment:
use `sectionHeightDesktop` instead of `heightSectionDesktop`, same for `heightSectionMobile`